semantics/cpp14/language/translation/decl/enum.k

```diff
@@ -80,23 +80,52 @@ module CPP-DECL-ENUM
                <enum-complete> false </enum-complete>
            ...</cppenum>)
 
-     syntax KItem ::= declareEnumerators(CPPEnumType, min: Int, max: Int, next: Int, nextType: CPPSimpleType, List, fixed: Bool, first: Bool)
-
-     rule declareEnumerators(_, _, _, Next::Int, NextType::CPPSimpleType, ListItem(Enumerator(_, NoExpression() => prv(Next, noTrace, type(NextType)))) _::List, _, _)
+     syntax KItem ::= declareEnumerators(CPPEnumType, min: Int, max: Int, next: Int, prevType: CPPSimpleType, List, fixed: Bool, first: Bool)
+
+     // First enumerator, fixed underlying type.
+     rule declareEnumerators(E::CPPEnumType, _, _, _, _, ListItem(Enumerator(_, NoExpression() => prv(0, noTrace, underlyingType(E)))) _::List, true, true)
+
+     // First enumerator, underlying type not fixed
+     rule (.K => UNSPEC("TDE2", "No initializer specified for the first enumerator")) ~> (
```

Comment:
Maybe "No initializer specified for the first enumerator, underlying type of enumerator is unspecified"?